src/runtime-rs/crates/resource/src/cpu_mem/initial_size.rs

```diff
@@ -19,6 +19,7 @@ use kata_types::{
 struct InitialSize {
     vcpu: u32,
     mem_mb: u32,
+    old_mem: u32,
```

Comment:
Suggest changing this name, which is a little confusing.

Reply:
fixed